src/app/article/article.component.html

```diff
@@ -0,0 +1,97 @@
+<div class="post-page" *ngIf="article">
```

Comment:
The angular way would be using async pipe

```suggestion
<div class="post-page" *ngIf="(article$ | async) as article">
```

```
export class ArticleComponent implements OnInit {
  article$: Obserbable<Article>;
```